src/bootstrap.js

```diff
@@ -4,7 +4,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 // for now we ship a stub, that can be upgraded as-needed after release
-function startup() {}
-function shutdown() {}
-function install() {}
-function uninstall() {}
+function startup() {} // eslint-disable-line no-unused-vars
```

Comment:
> I prefer not leave eslint comments to production code.

Me neither. However, `no-unused-vars` is not about the function being empty, it's about the function never getting called. `bootstrap.js` is not a Node module, so we cannot use `exported` here, so adding the comment is our only way to make eslint happy without turning `no-unused-vars` off for the entire file.